src/modules/shared/components/Hero/Hero.types.ts

```diff
@@ -0,0 +1,17 @@
+export interface HeroProps {
+	title: string;
+	description: string;
+	link: HeroLink;
+	image: HeroImage;
+}
+
+export interface HeroLink {
```

Comment:
Als je dit vaak ziet terugkomen mag je dat gerust in de shared types zetten als `ComponentLink` of iets dergelijk